src/collar/oc_titler.lsl

```diff
@@ -263,7 +315,7 @@ state active
         NukeOtherText();
         //llOwnerSay((string)llGetUsedMemory());
     }
-    timer(){
+/*    timer(){
```

Comment:
If we don't need a timer event anymore, let's remove it rather than leave this comment.